src/services/db/GitFileSystemService.ts

```diff
@@ -400,6 +402,49 @@ export default class GitFileSystemService {
     })
   }
 
+  getFileExtension(fileName: string): string {
+    const parts = fileName.split(".")
+    if (parts.length > 1) {
+      return parts[parts.length - 1]
+    }
+    return "" // No extension found
+  }
+
+  getMimeType(fileExtension: string): string {
+    switch (fileExtension) {
+      case "svg":
+        return "image/svg+xml"
+      case "ico":
+        return "image/vnd.microsoft.icon"
+      default:
+        return `image/${fileExtension}`
+    }
+  }
```

Comment:
Hey we do allow a bigger range of file types, eg. `.png`, `.jpeg`, etc. Are these taken into account as well?

Reply:
Yea they fall into the default case of image/{extension}

ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

Reply:
hmm yeah let's add these in, I think we might have overlooked these in the past